sirepo/sim_data/radia.py

```diff
@@ -75,6 +87,14 @@ def _find_obj_by_name(obj_arr, obj_name):
                 dm[m][f] = '0'
         cls._organize_example(data)
 
+    @classmethod
+    def sim_files_to_run_dir(cls, data, run_dir, post_init=False):
```

Comment:
The files do not necessarily exist when ```sim_data.support_files_to_run_dir()``` gets called by ```prepare_simulation()```, because geometryReport generates them when required, and all reports, including geometryReport, subsequently read them